wandb/plots/plot_definitions.py

```diff
@@ -716,3 +716,53 @@
   ]
 }
 '''
+
```

Comment:
What is this file for?  If this is just for reference let's turn it into a markdown file.

Reply:
yup this is just a reference for how we render the plots in vega.
Vega charts are ephemeral and also the definitions are only available to the creator, so I wanted to save the plot definitions in a more permanent place for anyone to reference.

Reply:
Let's still turn this into markdown.  It will render nicely in GH.

## heatmap/v1

```json
{
    "$schema": "https://vega.github.io/schema/vega-lite/v4.json",
    "padding": 5,
    "width": 500,
    "height": 500,
    "data":
      {
        "name": "${history-table:rows:x-axis,key}"
      },
    "title": {
      "text": {"value": ""}
    },
    "encoding": {
      "x": {
        "field": "x_axis",
        "type": "nominal",
        "axis": { "title": "" }
      },
      "y": {
        "field": "y_axis",
        "type": "nominal",
        "axis": { "title": "" }
      }
    },
    "layer": [
      {
        "mark": "rect",
        "encoding": {
          "color": {
            "field": "values",
            "type": "quantitative",
            "title": "Values",
            "scale": {
              "scheme": "tealblues"
            }
          }
        }
      },
      {
        "mark": "text",
        "encoding": {
          "text": {"field": "values", "type": "quantitative"}
        }
      }
    ]
}
```